khd/kernels/scattermoe/triton_implementation/kernels.py

```diff
@@ -113,8 +123,9 @@ def groupXtY_triton_kernel(
     stride_dwk,
     stride_dwn,
     expert_offsets_ptr,
-    K,
-    N,
+    K: tl.constexpr,
+    N: tl.constexpr,
+    E: tl.constexpr,
```

Comment:
looks like E is not used in this kernel, lets not pass this.

Reply:
`E` is used in autotune, i added it see above